tools/SetupFlow/DevHome.SetupFlow/ViewModels/MainPageViewModel.cs

```diff
@@ -103,12 +103,15 @@ public async Task StartConfigurationFileAsync(StorageFile file)
         }
     }
 
-    internal void StartAppManagementFlow(string query)
+    internal void StartAppManagementFlow(string query = null)
     {
-        _log.Information($"Launching app management flow for query:{query}");
+        _log.Information("Launching app management flow");
         var appManagementSetupFlow = _host.GetService<AppManagementTaskGroup>();
-        StartSetupFlowForTaskGroups(null, "App Search URI", appManagementSetupFlow);
-        appManagementSetupFlow.HandleSearchQuery(query);
+        StartSetupFlowForTaskGroups(null, "App Activation URI", appManagementSetupFlow);
+        if (!string.IsNullOrEmpty(query))
+        {
+            appManagementSetupFlow.HandleSearchQuery(query);

```

Comment:
`HandleSearchQuery` eventually handles the case of a null or empty string.  No need to this check.